apps/recovery-relay/lib/wallets/TON/index.ts

```diff
@@ -0,0 +1,106 @@
+import { Ton as BaseTon } from '@fireblocks/wallet-derivation';
+import { ConnectedWallet } from '../ConnectedWallet';
+import { TonClient, WalletContractV4 } from '@ton/ton';
+import { beginCell, Cell, fromNano } from '@ton/core';
+import { AccountData } from '../types';
+import { defaultTonWalletV4R2code } from './tonParams';
+import axios from 'axios';
+
+export class Ton extends BaseTon implements ConnectedWallet {
```

Comment:
`ConnectedWallet` is not used for memo, please see ADA or other LateInit wallets for memo usage and update

Reply:
I saw reference in the [docs](https://github.com/fireblocks/recovery/blob/01c4c1f482c5958688c38507758aca98422e5bf2/apps/recovery-relay/lib/README.md?plain=1#L18) to use the `LateInit` wallet only if a public RPC is not available, how does it affect working with a memo?